kroxylicious-krpc-plugin/src/main/java/io/kroxylicious/krpccodegen/main/KrpcGenerator.java

```diff
@@ -43,7 +43,7 @@
 import freemarker.template.Version;
 
 /**
- * Code generator driven by Apache Kafka message specifications definitions.
+ * Code generator driven by Apache Kafka&#174; message specifications definitions.
```

Comment:
Do we need to acknowledge the mark in _every_ source file that mentions Kafka?  

https://www.apache.org/foundation/marks/guide#naming says:

> The first and most prominent mentions must use the full form: "Apache Hadoop®" of the name for any individual usage (webpage, handout, slides, etc.)

Applying this to source files, I think we should limit it to:

* acknowledge the mark at the **program's entry point(s)** e.g. `io.kroxylicious.app.Kroxylicious`
* in javadoc for **public API** both at the package doc and class level.